build/linux/installer/conf/fluent-bit-common.conf

```diff
@@ -84,3 +84,12 @@
     Retry_Limit                     10
     TelemetryPushIntervalSeconds    300
     Match                           oms.container.*
+
+# TODO: Merge this with oms_output plugin as both of them are using the same plugin
```

Comment:
are we merging this with oms_output plugin?

Reply:
Removed the TODOs